Sources/SecureStore/KeyManagerService.swift

```diff
@@ -193,7 +196,9 @@ extension KeyManagerService {
             throw SecureStoreError.cantDecryptData
         }
 
-        guard let decryptedString = String(data: decryptData as Data, encoding: String.Encoding.utf8) else {
+        let dataToDecrypt: String? = String(decoding: decryptData as Data, as: UTF8.self)
```

Comment:
Same here.  The point of the swiftliint warning and these changes is that, if it is possible to use an initializer that will guarantee a non-nil return value, then you should use it.
If you have a look at the documentation for these initializers, you will see they are not necessarily applicable to all types of data.  However, the compiler is letting us know that the type of Data we are using passes the test, and so can guarantee intiialization.